spec/models/pickup_scheduler_spec.rb

```diff
@@ -54,4 +54,14 @@
       end
     end
   end
+
+  it "enrolls Donors in the Zone" do
```

Comment:
Where did the donor come from in this spec? Is it a factory association? What about explicitly creating it and then doing something like this:

```
expect(Donation.all.map(&:donor)).to eq [donor]
```

Does that still communicate what you want?